src/HomeWork6/ClassesHomeWork6/Account.java

```diff
@@ -0,0 +1,69 @@
+package HomeWork6.ClassesHomeWork6;
+
+import java.util.Arrays;
+
+public interface Account {
```

Comment:
why is it interface?

Reply:
because, I want create interface with nested class. I didn't knew that is wrong. Sorry